package.json

```diff
@@ -45,11 +48,13 @@
     "@playwright/test": "^1.27.1",
     "@svgr/webpack": "^6.2.1",
     "@tailwindcss/typography": "^0.5.4",
+    "@types/chance": "^1.1.3",
```

Comment:
This should be moved into dev dependencies

Reply:
Great catch ❤️

Reply:
We made a mistake, it is in devDependencies 😆